app/controllers/v1/benchmarks_controller.rb

```diff
@@ -3,6 +3,11 @@
 module V1
   # API for Benchmarks
   class BenchmarksController < ApplicationController
+    INCLUDES = {
+      nil => [],
```

Comment:
I'm all in for implementing this in the serializer, but find someone who cleans up that codebase first :sweat_smile:

Reply:
As discussed in slack, a constant is not really useful to have inherited behavior with slightly different data. You need to use some instance-based storage, like a method or instance variable. Since you don't have access to the controller's initializer, you can't define an instance variable. The only thing I can think to use is a method, as was originally implemented.